workflow/Snakefile

```diff
@@ -26,36 +27,40 @@ PHENOTYPE_WELLS = list(phenotype_samples_df["well"].unique())
 PHENOTYPE_TILES = list(phenotype_samples_df["tile"].unique())
 
 # TODO: unrestrict wells and tiles after testing
-SBS_WELLS = ["A1"]
+# SBS_WELLS = ["A1"]
 SBS_TILES = [1]
-# SBS_CYCLES = [1]
-PHENOTYPE_WELLS = ["A1"]
+# PHENOTYPE_WELLS = ["A1"]
 PHENOTYPE_TILES = [1]
 
-
 include: "targets/preprocess.smk"
 include: "rules/preprocess.smk"
 
-
 ALL_TARGETS = PREPROCESS_TARGETS_ALL
 
 if "sbs_process" in config:
-
+    # Initialize parameter search configurations if needed
+    if config["sbs_process"]["mode"] == "segment_sbs_paramsearch":
+        config = initialize_segment_sbs_paramsearch(config)
+        
+    # Include target and rule files
     include: "targets/sbs_process.smk"
     include: "rules/sbs_process.smk"
-
+    
     ALL_TARGETS += SBS_PROCESS_TARGETS_ALL
 
-
 if "phenotype_process" in config:
-
+    # Initialize parameter search configurations if needed
+    if config["phenotype_process"]["mode"] == "segment_phenotype_paramsearch":
+        config = initialize_segment_phenotype_paramsearch(config)
+        
+    # Include target and rule files
     include: "targets/phenotype_process.smk"
     include: "rules/phenotype_process.smk"
-
+    
     ALL_TARGETS += PHENOTYPE_PROCESS_TARGETS_ALL
 
 
 # Define the target files for the workflow
 rule all:
     input:
-        ALL_TARGETS,
+        ALL_TARGETS,
```

Comment:
```suggestion
        ALL_TARGETS,

```

Newline convention

Reply:
Changed [here](https://github.com/cheeseman-lab/brieflow/pull/23/commits/15adce4acec642cbea13c60b6ec4fc1b888c1cf2)--are you able to tell this from lint ing? Doesn't show up for me.